### PR TITLE
OTR: Updating opening section

### PR DESCRIPTION
**Pull request by tommycollison**
As per #74. 

- Shortening the introduction, splitting most of the information into "about OTR" 
- Adding a section differentiating Google off the record and cypherpunks OTR.
- Changing Micah's "I recommend" to "FPF recommends."

### CODE REVIEW

**Comment by conorsch** [comment]
### Headings got funky:
* OTR has duplicated headings
* headings got bigger (## -> # and ### -> ## in markdown)—why? I think they should be left the same.

### Google OTR
Google's OTR "turns off logging"—*for whom?* Almost sounds like Google is disabling logging server-side, which we really can't know. In fact, Google [explains OTR](https://support.google.com/chat/answer/29291?hl=en) like so:

> Chats that have been taken off the record aren't stored in your Gmail chat history, or in the Gmail chat history of the person you're chatting with. 

So logging via a desktop client is still permissible, and we don't have any reason to believe Google isn't still reading the chat logs.

File: encryption_works.md [hunk 1]
```diff
@@ -120,21 +120,31 @@ When Snowden [was answering questions on Guardian's website](http://www.guardian
...
+Some instant message chat programs, such as Google Talk, support adding a layer of encryption to conversations. With this end-to-end encryption software, called "Off the Record" (OTR), you can have secure chats using Google Talk and Facebook chat without Google or Facebook ever being able to see the contents of your conversation. 
 
-OTR is used for two things: **encrypting the contents** of real-time instant message conversations and **verifying the identity** of people that you chat with. Identity verification is extremely important and something that many OTR users neglect to do. While OTR is much more user friendly that other forms of public key encryption, if you wish to use it securely you still need to understand how it works and what attacks against it are possible.
```
Comment (trevortimm) [inline encryption_works.md]: Most of the original intro is now at the bottom, when it belongs up top. People will want to know what they are signing up for and the purposes of it before they actually go through the steps to install it.

File: encryption_works.md [hunk 2]
```diff
@@ -120,21 +120,31 @@ When Snowden [was answering questions on Guardian's website](http://www.guardian
...
 
-### OTR Clients
+If you haven't yet updated from Google Talk to Hangouts, the feature is confusingly also called "off the record." 
 
-To use OTR you'll need to download software. If you use Windows you can download and install [Pidgin](https://pidgin.im/) and separately the [OTR plugin](http://www.cypherpunks.ca/otr/). If you use GNU/Linux you can install the pidgin and pidgin-otr packages. You can read through [documentation on how to set up your Pidgin accounts with OTR](http://www.cypherpunks.ca/otr/index.php#docs). If you use Mac OS X you can download and install [Adium](http://adium.im/), which is a free software chat client that includes OTR support. You can read the [official documentation](http://adium.im/help/pgs/AdvancedFeatures-OTREncryption.html) on how to get set up with OTR encryption with Adium.
```
Comment (trevortimm) [inline encryption_works.md]: I say we keep all the Gmail stuff you put in, but also what Micah has about OTR clients and Jabber servers (maybe stealing some of his language from his recent Intercept piece and then linking to the rest). For the reasons stated above, it's more important to include jabber than it is gchat for metadata and anonymity purposes purposes. 

Basically, this section looks good, it's just a bit out of order.

**Comment by garrettr** [comment]
Rebased and merged into master, see https://github.com/freedomofpress/encryption-works/pull/86#issuecomment-130828377 for details.

**Pull request closed without merging**